### PR TITLE
Add locales for copy/paste

### PR DESCRIPTION
**Pull request by AliyanH**
For https://github.com/Maps4HTML/Web-Map-Custom-Element/pull/693

### CODE REVIEW

**Comment by AliyanH** [comment]
Hey @Malvoz would you be able to look at the Swedish translation? Would be a great help, thank you very much!

File: src/_locales/sv/messages.json [hunk 1]
```diff
@@ -28,6 +28,9 @@
   "cmCopyMapML": {
     "message": "Kopiera MapML"
   },
+  "cmPasteLayer": {
+    "message": "Pasta"
```
Comment (Malvoz) [inline src/_locales/sv/messages.json:32]:
```suggestion
    "message": "Klistra in"
```

File: src/_locales/sv/messages.json [hunk 2]
```diff
@@ -38,6 +41,9 @@
   "lmZoomToLayer" : {
     "message": "Zooma Till Lager"
   },
+  "lmCopyLayer" : {
+    "message": "Kopiera Lagret"
```
Comment (Malvoz) [inline src/_locales/sv/messages.json:45]:
```suggestion
    "message": "Kopiera Lager"
```

**Review by prushforth — COMMENTED** [review]
👍

**Pull request merged**